### DISCUSSION TITLE
some big problems on phone！！！（English+Chinese）

### DISCUSSION THREAD

**Discussion by adtcqd** [HasangerGames/suroi#332] [General]
I came from China so ↓is written by translate APP，if you can read Chinese，pleaseread the Chinese part，thanks you！

English part（Chinese part is in the↓↓↓）：
Hello, I come from China. I often play this game with my mobile phone, and I will also play this game with my classmates, but we encounter many problems! 1. How does the mobile version put items on the ground for teammates to pick up? 2. Although I chose the Asian region, I still have a delay of 100+. However, if I use the https://suroi.fpsgo.net/ game, although it is version 0.15, the delay is very low. Why can't I merge these two servers? 3. The logical error in throwing grenades in the mobile version is not to loosen the rocker and throw. If forced operation, the Grenade may be thrown to the right! Thank you for reading my poor English, which is the limit of my English level:) Leave the rest to the translation software! ! !

中文部分（更加准确 但可能对您造成阅读障碍）：
你好 我来自中国 我常常使用手机玩这个游戏 还会和全班同学一起玩这个游戏 但是我们遇到很多问题！
1.手机版如何把物品放在地上给队友捡起？
2.虽然我选择了亚洲区 但是我依旧有100+的延迟 然而如果我用https://suroi.fpsgo.net/游戏 虽然这是0.15版本 但是延迟很低 请问为什么不能把这两个服务器合并？
3.手机版投掷手雷时操作逻辑错误 并不是松开摇杆投掷 如果强行操作可能导致手雷固定向右投掷！
4.谢谢你阅读我不流畅的英语 这是我英语水平的极限:) 其他的就交给翻译软件吧！！！

**Comment by emeraldn30** [comment]
The asia servers of https://suroi.io are located in Vietnam. https://suroi.fpsgo.net/ is closer to where you live than the https://suroi.io servers, so it has less latency.

https://suroi.io 的亚洲服务器位于越南
https://suroi.fpsgo.net/ 比 https://suroi.io 服务器更靠近您居住地，因此延迟更少

**Reply by adtcqd** [reply]
谢谢你的回复和中文回复 请问为什么不把https://suroi.fpsgo.net的服务器升级为18.0呢:)
Thank you for your reply and Chinese reply. Why not upgrade the server in https://suroi.fpsgo.net to 18.0?:)

**Reply by bien-star** [reply]
suroi fpsgo is presumably an independetly hosted site. If you could get into contact with the people who host it, they would probobly be glad to update it

**Reply by juni-purr** [reply]
Emerald your chinese sucks
